SINK.md

```diff
@@ -0,0 +1,39 @@
+# Sink Ruter
+
+This contains transaction data for the old 'Troms Billett' app
+develped by Ruter as part of the now dead KK 1.0 collaboration.
+
+The transaction in this app stood for the majority of our
```

Comment:
```suggestion
The transactions in this app stood for the majority of our
```

Reply:
I think plural is correct here. "The app" as a concept was generating numerous transactions …

Reply:
I suggested plural, so we should commit that change if we think it is correct :)

Reply:
Christ almighty … sorry 'bout that. Fixing now …